src/api/getAttendee.ts

```
@@ -0,0 +1,23 @@
+		`https://fe-adv-project-together-dallaem.vercel.app/7/gatherings/${id}/participants`,
+		{
+			method: 'GET',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		},
+	);
+
+	if (!response.ok) {
+		const error = await response.json();
+		console.error(error);
```

ESLint

[error] Unexpected console statement.

(no-console)